### PR TITLE
Update missing Farsi translations.

### PR DESCRIPTION
**Pull request by mohebifar**
Fix #3281 conflicts with the master `en` version.

### CODE REVIEW

File: Resources/translations/SonataAdminBundle.fa.xliff [hunk 1]
```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0"?>
+<؟xml version="1.0"؟>
```
Comment (Soullivaneuh) [inline Resources/translations/SonataAdminBundle.fa.xliff]: This should not be translated. Please keep `?` character.

**Comment by OskarStark** [comment]
Can you please squash your commits?

**Comment by OskarStark** [comment]
And rebase please @mohebifar

**Comment by greg0ire** [comment]
ping @mohebifar

**Comment by core23** [comment]
Please reabse @mohebifar

**Comment by greg0ire** [comment]
@mohebifar ?

**Comment by greg0ire** [comment]
@OskarStark @core23 : please close this in favor of #3710

**Pull request closed without merging**